test/cpp-tableau-loader/src/protoconf/hero_conf.pc.h

```diff
@@ -15,7 +16,7 @@ namespace tableau {
 class HeroConf : public Messager {
  public:
   static const std::string& Name() { return kProtoName; }
```

Comment:
Base class `Messager` already implemented  the same func:
```cpp
static const std::string& Name() { return kProtoName; }
```
So there is no need to re-implement it again.